zamba/models/cnnensemble_model.py

```diff
@@ -94,7 +94,9 @@ def predict(self, file_names, resample=True, seperate_blank_model=True):
                 classes
         """
         processed_paths = self.preprocess_videos(file_names, resample=resample)
-        valid_videos = list(processed_paths.values())
+
+        # exclude videos where output path doesn't exist (e.g. videos that can't be resampled)
+        valid_videos = [v for v in processed_paths.values() if v.exists()]
```

Comment:
Should `invalid_videos` below be started with the ones that don't exist? Or, is there another way to surface that to the user?

Reply:
they get added to the predictions as nans [here](https://github.com/drivendataorg/zamba/blob/master/zamba/models/cnnensemble_model.py#L164-L170)  b/c they're in processed paths which is why the tests pass without starting `invalid_videos` with the vids that don't exist. but am very open to starting invalid vids with the ones that don't exist to be extra explicit

Reply:
i have a preference for starting invalid vids with the paths that don't exist so that it's easy to see them getting set to nan [here](https://github.com/drivendataorg/zamba/blob/master/zamba/models/cnnensemble_model.py#L149-L159) rather than in a hidden way w/ the index. will make that change

Reply:
Ah, I had assumed they had to [be in here to work](https://github.com/drivendataorg/zamba/blob/master/zamba/models/cnnensemble_model.py#L154) and that line would throw an error if there was a shape mismatch 🤷